staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -295,7 +302,41 @@ def update(self):
         self.updatePlaylistDetails()
         self.updateCoverArtFromURLs()
 
-  ##############################################################################
+        #trigger a hdmi command, if a controller has changed the power status or the volume
+        if constants.TURNONAVR == 'true' and constants.CECSUPPORT == 'true':
+          powerstate = self.player.getpowerstate()
+          if powerstate == 0 and constants.HDMIPOWER == "true":
+            if constants.TURNOFFTV == "true":
+               tv.standby()
+               pass
+            if constants.TURNONAVR == "true":
+               avr.standby()
+            constants.HDMIPOWER = "false"
+          elif powerstate != 0 and constants.HDMIPOWER == "false":
+            if constants.TURNONAVR == "true":
+               avr.power_on()
+               avr.set_av_input(int(constants.AVRINPUT))
+            constants.HDMIPOWER = "true"
+          else:
+            pass
+
+        if constants.ADJUSTVOL == "true" and constants.CECSUPPORT == 'true':
+          vol1 = self.player.getVolume()
+          xbmc.sleep(1000)
```

Comment:
Well, I have expected this question. Maybe it is not the best solution but it works :-) Here I want to find out if the controller app has turned up or turned down the volume. After putting the system to sleep the second call for volume delivers another value than the first one.